### ISSUE TITLE
Flickering in Nushell when used with Zellij and WezTerm

### ISSUE THREAD

**Issue by yoav-lavi** [zellij-org/zellij#2794]
When e.g. displaying completions over a certain length, the output form Nushell will flicker under Zellij when used in WezTerm.

This does not happen when using Nushell directly under WezTerm (see below)

With Zellij:

https://github.com/zellij-org/zellij/assets/14347895/7d6890b1-82fa-4764-acfe-6939e8a9145e

Without Zellij:

https://github.com/zellij-org/zellij/assets/14347895/ba8df253-a58e-4bbc-a867-b5d8f3c03158

I'm using Zellij WezTerm and Nushell under Nix (darwin).

Possibly Related:

https://github.com/nushell/nushell/issues/9279
https://github.com/nushell/nushell/issues/6214

Log:

```log
WARN   |zellij_utils::input::layo| 2023-09-18 14:24:35.448 [main      ] [zellij-utils/src/input/layout.rs:962]: Failed to read swap layout file: /Users/yoavlavi/.config/zellij/layouts/layout.swap.kdl. Error: Os { code: 2, kind: NotFound, message: "No such file or directory" } 
INFO   |zellij_client            | 2023-09-18 14:24:35.451 [main      ] [zellij-client/src/lib.rs:152]: Starting Zellij client! 
INFO   |zellij_server            | 2023-09-18 14:24:35.458 [main      ] [zellij-server/src/lib.rs:245]: Starting Zellij server! 
INFO   |zellij_server            | 2023-09-18 14:24:35.515 [main      ] [zellij-server/src/lib.rs:913]: Compiling plugins using Cranelift 
INFO   |zellij_server::plugins   | 2023-09-18 14:24:35.515 [wasm      ] [zellij-server/src/plugins/mod.rs:137]: Wasm main thread starts 
INFO   |zellij_server::plugins::p| 2023-09-18 14:24:35.529 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:487]: Loaded plugin 'compact-bar' from cache folder at '/Users/yoavlavi/Library/Caches/org.Zellij-Contributors.Zellij' in 5.527375ms 
INFO   |zellij_server::plugins::p| 2023-09-18 14:24:35.529 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:487]: Loaded plugin 'compact-bar' from cache folder at '/Users/yoavlavi/Library/Caches/org.Zellij-Contributors.Zellij' in 6.388667ms 
INFO   |zellij_server::plugins::p| 2023-09-18 14:24:35.535 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:487]: Loaded plugin 'compact-bar' from cache folder at '/Users/yoavlavi/Library/Caches/org.Zellij-Contributors.Zellij' in 11.513541ms 
INFO   |zellij_server::plugins::p| 2023-09-18 14:24:35.538 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:487]: Loaded plugin 'compact-bar' from cache folder at '/Users/yoavlavi/Library/Caches/org.Zellij-Contributors.Zellij' in 14.867667ms 
INFO   |zellij_server::plugins::p| 2023-09-18 14:24:35.546 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:487]: Loaded plugin 'compact-bar' from cache folder at '/Users/yoavlavi/Library/Caches/org.Zellij-Contributors.Zellij' in 22.85325ms 
INFO   |zellij_server::plugins::p| 2023-09-18 14:24:35.549 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:487]: Loaded plugin 'compact-bar' from cache folder at '/Users/yoavlavi/Library/Caches/org.Zellij-Contributors.Zellij' in 26.676125ms 
INFO   |zellij_server::plugins::p| 2023-09-18 14:24:35.550 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:487]: Loaded plugin 'compact-bar' from cache folder at '/Users/yoavlavi/Library/Caches/org.Zellij-Contributors.Zellij' in 23.19775ms 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.630 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105, 47, 100, 101, 118, 47, 103, 114, 97, 102, 98, 97, 115, 101]] 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.634 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105, 47, 46, 99, 111, 110, 102, 105, 103, 47, 100, 97, 114, 119, 105, 110]] 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.638 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105, 47, 100, 101, 118, 47, 103, 114, 97, 102, 98, 97, 115, 101]] 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.640 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105, 47, 46, 99, 111, 110, 102, 105, 103, 47, 100, 97, 114, 119, 105, 110]] 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.648 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105, 47, 100, 101, 118, 47, 97, 112, 105]] 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.670 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105, 47, 100, 101, 118, 47, 97, 112, 105]] 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.676 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105, 47, 100, 101, 118, 47, 112, 97, 116, 104, 102, 105, 110, 100, 101, 114]] 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.677 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105]] 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.678 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105]] 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.683 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105, 47, 100, 101, 118, 47, 112, 97, 116, 104, 102, 105, 110, 100, 101, 114]] 
WARN   |zellij_server::panes::gri| 2023-09-18 14:24:35.696 [screen    ] [zellij-server/src/panes/grid.rs:2385]: Unhandled osc: [[55], [102, 105, 108, 101, 58, 47, 47, 87, 111, 114, 107, 45, 77, 97, 99, 98, 111, 111, 107, 45, 80, 114, 111, 46, 108, 111, 99, 97, 108, 47, 85, 115, 101, 114, 115, 47, 121, 111, 97, 118, 108, 97, 118, 105]] 
INFO   |zellij_client            | 2023-09-18 14:25:12.360 [main      ] [zellij-client/src/lib.rs:479]: Bye from Zellij! 
INFO   |zellij_server::plugins::w| 2023-09-18 14:25:12.360 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 1 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.361 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           10,
           [
               13,
               10,
           ],
       ) 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.362 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           1,
           [
               13,
               10,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
           ],
       ) 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.363 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           8,
           [
               13,
               10,
           ],
       ) 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.363 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           6,
           [
               13,
               10,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
           ],
       ) 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.363 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           0,
           [
               13,
               10,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
           ],
       ) 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.364 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           7,
           [
               13,
               10,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
           ],
       ) 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.365 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           5,
           [
               13,
               10,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
           ],
       ) 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.365 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           4,
           [
               13,
               10,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
           ],
       ) 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.366 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           3,
           [
               13,
               10,
           ],
       ) 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.366 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           9,
           [
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
           ],
       ) 
ERROR  |zellij_utils::errors::not| 2023-09-18 14:25:12.366 [async-std/runti] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: async-std/runtime
             location: At zellij-server/src/pty.rs:700:30
             message: Program terminates: a fatal error occured

Caused by:
    0: failed to spawn terminals for layout
    1: failed to listen for bytes from PTY
    2: failed to async-send to screen
    3: failed to send message to screen
    4: Originating Thread(s)
       
    5: failed to send message to channel: PtyBytes(
           2,
           [
               13,
               10,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
               27,
               91,
               51,
               48,
               109,
               27,
               40,
               66,
               27,
               91,
               109,
           ],
       ) 
INFO   |zellij_server::plugins::w| 2023-09-18 14:25:12.368 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 4 
INFO   |zellij_server::plugins::w| 2023-09-18 14:25:12.369 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 5 
INFO   |zellij_server::plugins::w| 2023-09-18 14:25:12.371 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 6 
INFO   |zellij_server::plugins::w| 2023-09-18 14:25:12.373 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 2 
INFO   |zellij_server::plugins::w| 2023-09-18 14:25:12.374 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 0 
INFO   |zellij_server::plugins::w| 2023-09-18 14:25:12.375 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 3 
INFO   |zellij_server::plugins   | 2023-09-18 14:25:12.377 [wasm      ] [zellij-server/src/plugins/mod.rs:339]: wasm main thread exits 
```

**Basic information**

`zellij --version`: `zeelij 0.38.2`
`stty size`: `97 318`
`uname -av` or `ver`(Windows): `Darwin {{HOSTNAME}} 22.6.0 Darwin Kernel Version 22.6.0: Wed Jul  5 22:22:05 PDT 2023; root:xnu-8796.141.3~6/RELEASE_ARM64_T6000 arm64 arm Darwin`

List of programs you interact with as, `PROGRAM --version`: output cropped meaningful, for example:

`wezterm --version`: `wezterm 20230712-072601-f4abf8fd`
`nu --version`: `0.84.0`

**Comment by imsnif** [comment]
Hey, thanks for the report!

Zellij - still being pre 1.0 - is unfortunately not a 60fps application. While we have optimized many parts of the application to often be even faster than when running in a bare terminal (eg. dumping large amounts of text into a terminal), other parts such as these still need some attention.

Due to the limited number of working hands though, these issues are not a priority (eg. over things like bugs, core features such as session resurrection, a windows version and a web client).

Which is to say - this is the way things are going to be for the near future. I plan on addressing these before we release a 1.0.

**Comment by pallaswept** [comment]
I'm seeing this same behaviour using pw-top (pipewire's monitoring app). Every second, it updates, and flickers. I'm afraid that's a deal-breaker for me, it's a migraine and epilepsy trigger. The front page should probably carry a health warning until you've got this sorted out.

**Comment by imsnif** [comment]
@pallaswept - while I can empathize and am sorry this causes you discomfort, this is not necessarily the same issue and until I see a reproduction with logs as mentioned in the issue template, I would also not necessarily say this is a Zellij issue.

People have a tendency to pile up on these issues, making them far less likely to be addressed due to the chaos this causes. If you'd like to help out, please open a separate issue with a reproduction as mentioned in the issue template - otherwise I would encourage you to use or not use whichever software makes you happy.

**Comment by pallaswept** [comment]
> @pallaswept - while I can empathize and am sorry this causes you discomfort, this is not necessarily the same issue 

Totally, my bad... but it's not a matter of discomfort, it's a health risk, and for some, it can be life-threatening.

> and until I see a reproduction with logs as mentioned in the issue template, I would also not necessarily say this is a Zellij issue.

Given that this exclusively occurs in zellij,not tmux or anything else, and happens in zellij, across foot, kitty, and konsole, I'm pretty confident that zellij is the issue here. I've reproduced it (while avoiding looking at the screen) and created a new issue here: https://github.com/zellij-org/zellij/issues/2828 But there's nothing in the logs of note, I'm afraid.

> People have a tendency to pile up on these issues, making them far less likely to be addressed due to the chaos this causes. If you'd like to help out, please open a separate issue with a reproduction as mentioned in the issue template -

You're quite right, so I have done so.

>  otherwise I would encourage you to use or not use whichever software makes you happy.

Yeh I'm going to have to use tmux, which doesn't make me happy, I preferred zellij (this bug aside). It's not a matter of happiness, it's a matter of physical health. 

Seizures are serious man, if you're not familiar with it (you're a programmer not a Doctor, no shame in that) you should look it up in your favourite search engine so you understand what's at play here. Since you know this app has flickering issues and now you know it's for real dangerous, empathy is nice.... but a warning is appropriate. "I'm really sorry about your legs" are polite words but don't make up for it if you put a big hole in the ground and don't put a fence around it, ya know what I mean? 😆

**Comment by imsnif** [comment]
I am very sorry for your experience and will do my best to address the issue and understand it better.

**Comment by brianmego** [comment]
@imsnif, I would be interested in taking a look at this. If you have an inkling of the area of code this might reside in, or what logs to pay attention to, that would be helpful.

**Comment by pallaswept** [comment]
> @imsnif, I would be interested in taking a look at this. If you have an inkling of the area of code this might reside in, or what logs to pay attention to, that would be helpful.

This might have been fixed by https://github.com/zellij-org/zellij/pull/2977 as was #2828 ?

**Comment by brianmego** [comment]
Well, that's a pretty good inkling! Did it indeed fix the problem, @pallaswept ? I see it's merged into main, but perhaps that hasn't been cut as a release yet?

**Comment by pallaswept** [comment]
> Well, that's a pretty good inkling! Did it indeed fix the problem, @pallaswept ? I see it's merged into main, but perhaps that hasn't been cut as a release yet?

Good point, it hasn't actually made it into a release yet. The latest release was November 29, but this patch wasn't merged until December 3... I'm not sure if it actually fixed it, I haven't actually tested it (I'm not using a multiplexer at all at present, just waiting on a few other feature requests to make it all work together), but I did look at the code and I'm sure that for any terminal that supports synced output, that will do the job.

I guess, to be really sure, I/we would have to build from source, or wait for the next release of zellij... But I'm pretty confident that change should do the trick, so I was going to just wait it out. If you're a dev who was willing to try and fix this up, I guess I don't need to offer my help building from source if you want to test it, but the offer is there nonetheless, should you need it :)

**Comment by darrenburns** [comment]
This may be fixed by https://github.com/zellij-org/zellij/pull/3884.

**Comment by imsnif** [comment]
I just want to emphasize that I did not intend to close this issue. Indeed, I make it my policy not to close issues at all in this repo (see main README). This was auto closed by GitHub due to wording in the linked PR - this can be re-opened or closed at the discretion of the issue creator.
